### DISCUSSION TITLE
Login loop after authentification app integration

### DISCUSSION THREAD

**Discussion by CheRRy1975** [alandtse/alexa_media_player#2619] [Show and tell]
Here is something I found in the WWW...
This helped me with this problem!

"Through trial and error trying to figure out the flow sequence, I was able to trick it and get to the OTP window.
After you've cycled through name, password and back to name, select the option Create Account New to Amazon?
Fill in a first & last name (anything at all as not going to matter so "Q W" is quick and easy to type!)
Amazon will then inform you that an account with your email already exists.
At that point re-select "Sign in Already a customer?" and proceed with the email and password windows and then the OTP window should appear. By that point though, the integration generated OTP has likely expired but you can just enter your current app generated OTP code and it should work.
If it returns to the integration add window and it just sits and spins and never progresses, that likely means the proxy listening session has expired and you'll have to repeat the process."

**Comment by dpalczewski** [comment]
This seems to have worked for me, once log in just give it a restart and it will lead all the devices. Thanks, I was about to give up on this thing.
